CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing to the Kalispell Software Crafters GitHub Pages Website
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
```

Comment:
I'm expecting GitHub will display these icons correctly but my VS Code screenshot didn't do that.

Reply:
Looks like they work!
https://github.com/kalispell-software-crafters/kalispell-software-crafters.github.io/blob/feature/add-contribution-guide/CONTRIBUTING.md

Reply:
Really like the contributing guide from Atom. I appreciate that you put a section in for our code of conduct, too. I approve the merge